lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -342,12 +342,10 @@ def gem_version_compare(first, second)
                      curl --cacert $SSL_CA_CERT https://smart-proxy.example.com:8443"
           end
           def foreman_server_ca_cert
-            if File.exist?(Setting[:ssl_ca_file])
-              File.read(Setting[:ssl_ca_file])
-            else
-              msg = N_("SSL CA file not found, check the 'SSL CA file' in Settings > Authentication")
-              raise Foreman::Exception.new(msg)
-            end
+            File.read(Setting[:ssl_ca_file])
```

Comment:
I do think that `Setting[:ssl_ca_file].blank?` deserves its own message. Perhaps
```suggestion
            raise N_('No CA file set') if Setting[:ssl_ca_file].blank?

            File.read(Setting[:ssl_ca_file])
```

My reasoning is that as a user, it's surprising to see it trying to read a file without a name and see an error for that.